src/uiSelectDirective.js

```diff
@@ -161,7 +161,7 @@ uis.directive('uiSelect',
         });
 
         //Automatically gets focus when loaded
-        if (angular.isDefined(attrs.autofocus)){
+        if (angular.isDefined(attrs.autofocus) && $parse(attrs.autofocus)()) {
```

Comment:
Isn’t attrs.$observe not better to use instead of parse?

Reply:
It does not need to $observe for changes, just the initial focus when loaded